tests/tt_eager/python_api_testing/unit_testing/misc/test_topk.py

```diff
@@ -67,7 +67,16 @@ def run_topk_test(N, C, H, W, k, dtype, device):
         # "FLOAT32",
     ],
 )
-@pytest.mark.parametrize("N, C, H, W, k,", ((1, 1, 32, 64, 32),))
+@pytest.mark.parametrize(
```

Comment:
do we have ttnn support for this?

Reply:
Up until this PR this OP was hardcoded for 32,64 so I didn't find it generic enough. In the next PR I make related to this OP I'm going to add multi core support, and I'll then add it to ttnn with a test.